data_steward/cdr_cleaner/cleaning_rules/base_cleaning_rule.py

```diff
@@ -268,49 +268,57 @@ def __validate_arguments(self):
                                format(clazz))
                     raise TypeError(message)
 
-    def get_depends_on_classes(self):
+    @property
```

Comment:
this is neat, I've never seen/used `@property` before and I looked it up!

Reply:
great!  new things for us both!